project_event/views/activity_template_view.xml

```diff
@@ -8,8 +8,7 @@
             <form string="Activity Template">
                 <sheet>
                     <div>
-                        <h1>
-                            <field name="name"/>
+                        <h1><field name="name"/>
```

Comment:
@eilst indentation

Reply:
I used the automatic format, I think it's right